schematic/utils/general.py

```diff
@@ -26,3 +27,30 @@ def unlist(_list):
         return _list[0]
     else:
         return _list
+
+def get_dir_size(path='.'):
```

Comment:
Nit (nitpicky): I don't think this needs to be an key argument, it can probably be positional and required when you call this function.

get_dir_size(path)